src/app/app.component.html

```diff
@@ -129,6 +129,13 @@
       </mat-list-item>
     </mat-nav-list>
 
+    <mat-nav-list>
+      <mat-list-item (click)="composeBugReport()" matTooltip="Compose an email to runbox support">
```

Comment:
Is there any reason this is its own nav-list, rather than another list-item in the previous nav-list?

Reply:
Only cos that one had an ngIf attached, I guess we could move that to the nav-list-item